src/Stryker.Core/Stryker.Core/Mutants/MutantPlacer.cs

```diff
@@ -1,12 +1,28 @@
 ﻿using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using System.Collections.Generic;
+using System.IO;
 
 namespace Stryker.Core.Mutants
 {
     public static class MutantPlacer
     {
+        private static readonly string helper;
+
+        static MutantPlacer()
+        {
+            using (var stream =
```

Comment:
Why not use:

``` csharp
string helper = File.ReadAllLines("Stryker.Core.Mutants.ActiveMutantHelper.cs");
```

?

Reply:
The file is embedded as an embedded resource. I don't how to use ReadAllLines for those. 
I am open to alternative strategies, if any.

Reply:
We can create a static helper for this in the testinghelpers dir? That way this part of our code looks cleaner and makes reuse for embedded files possible. But the current way is also ok for me.

Reply:
Can you share a link to the corresponding github folder? I am afraid I am not able to find this dir.

Reply:
This is also ok for now, when this functionallity is needed again for a new feature we'll move it to the helpers dir.